requirements.txt

```diff
@@ -11,7 +11,7 @@ Jinja2==3.1.2
 loguru==0.6.0
 mypy==1.0.0
 netifaces==0.11.0
-numpy==1.21.6
+numpy
```

Comment:
numpy is imported to speed up rendering on the TFT display, as recommended by https://github.com/adafruit/Adafruit_CircuitPython_RGB_Display . it's only used a small handful of places here and without a version bound; validate all versions of numpy haven't substantially changed the functions used here: https://github.com/adafruit/Adafruit_CircuitPython_RGB_Display/blob/main/adafruit_rgb_display/rgb.py and also that the TFT display works well.

Reply:
The only code that uses numpy in the Adafruit library is here: https://github.com/adafruit/Adafruit_CircuitPython_RGB_Display/blob/8cfb0db7d36eb46ef1e2a575b03f5e630bfc2c94/adafruit_rgb_display/rgb.py#L68

The functions & methods in question are:
```
numpy.array()
numpy.dstack().flatten().tolist()
```

On first brush with the documentation, these seem well supported (indeed, foundational) from 1.21 thru 1.26, the latest numpy release.

This still probably deserves a run on a box with a TFT display to validate.

Reply:
I validated this functions on a box with a TFT display.